tools/run_tests/performance/loadtest_config.py

```diff
@@ -112,7 +112,7 @@ def gen_run_indices(runs_per_test: int) -> Iterable[str]:
         yield ''
         return
     prefix_length = len('{:d}'.format(runs_per_test - 1))
-    prefix_fmt = '{{:{:d}d}}'.format(prefix_length)
+    prefix_fmt = '{{:0{:d}d}}'.format(prefix_length)
```

Comment:
looks cryptic, but if you tested this works the way it should, I'm fine with that.
- Why is it important for the "prefix" to be always the same length?
- AFAIR the job number get appended to the name as suffix, not as a prefix?

Reply:
* It is same length to ensure the tests sort correctly with `kubectl get` and the like.
* This function was split off from one where it was a prefix. Renamed `prefix` to `index` so it makes sense in the current code.